tools/clang/test/SemaHLSL/rayquery-decls-errors-TU-multiple.hlsl

```diff
@@ -0,0 +1,41 @@
+// RUN: %dxc -T vs_6_5 -E main -verify %s
+
+// tests that RAYQUERY_FLAG_ALLOW_OPACITY_MICROMAPS usage will emit
+// one warning for each incompatible availability attribute decl,
+// when the compilation target is less than shader model 6.9.
+
+namespace MyNamespace {
+  // expected-warning@+1{{potential misuse of built-in constant 'RAYQUERY_FLAG_ALLOW_OPACITY_MICROMAPS' in shader model vs_6_5; introduced in shader model 6.9}}
+  static const int badVar = RAYQUERY_FLAG_ALLOW_OPACITY_MICROMAPS;
+}
+
+// expected-warning@+1{{potential misuse of built-in constant 'RAYQUERY_FLAG_ALLOW_OPACITY_MICROMAPS' in shader model vs_6_5; introduced in shader model 6.9}}
+groupshared const int otherBadVar = RAYQUERY_FLAG_ALLOW_OPACITY_MICROMAPS;
+
+int retNum(){
+  // expected-warning@+1{{potential misuse of built-in constant 'RAYQUERY_FLAG_ALLOW_OPACITY_MICROMAPS' in shader model vs_6_5; introduced in shader model 6.9}}
+  return RAYQUERY_FLAG_ALLOW_OPACITY_MICROMAPS;
```

Comment:
Maybe one more copy of this function that isn't called to verify that it doesn't have a diagnostic?